src/Task/Git/CommitMessage.php

```diff
@@ -343,7 +343,7 @@ private function checkTypeScopeConventions($context)
             $scopesPattern = '(:\s|(\(' . $scopes . '\)?:\s))';
         }
 
-        $rule = '/^' . $typesPattern . $scopesPattern . $subjectPattern . '|' . $mergePattern . '/';
+        $rule = '/^(fixup!\s|squash!\s){0,1}' . $typesPattern . $scopesPattern . $subjectPattern . '|' . $mergePattern . '/';
```

Comment:
```suggestion
        $rule = '/^(?:(?:fixup|squash)!\s)?' . $typesPattern . $scopesPattern . $subjectPattern . '|' . $mergePattern . '/';
```

or 

```suggestion
        $rule = '/^(?:(?:fixup|squash)! )?' . $typesPattern . $scopesPattern . $subjectPattern . '|' . $mergePattern . '/';
```

as after the fixup / squash keywords, it is a space IIRC.